packages/measure/src/measure-renders.tsx

```diff
@@ -69,6 +78,19 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
     runResults.push({ duration, count });
   }
 
+  if (
+    jsonRepresentations.length >= minRecordToCompare &&
+    jsonRepresentations.every((json) => json === jsonRepresentations.at(0))
```

Comment:
The check should compare each pair of subsequent renders results, not each vs the first one

Reply:
Fixed with using dfs algorithm to compare saved jsons